pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -173,25 +169,23 @@ func reconcileObjectMeta(oldMeta *metav1.ObjectMeta, newMeta metav1.ObjectMeta)
 	return
 }
 
-func (c *Reconciler) reconcileService(el *v1alpha1.EventListener) error {
+func (r *Reconciler) reconcileService(logger *zap.SugaredLogger, el *v1alpha1.EventListener) error {
 	service := &corev1.Service{
 		ObjectMeta: generateObjectMeta(el),
 		Spec: corev1.ServiceSpec{
 			Selector: GenerateResourceLabels(el.Name),
 			Type:     el.Spec.ServiceType,
-			Ports: []corev1.ServicePort{
-				{
-					Name:     eventListenerServicePortName,
-					Protocol: corev1.ProtocolTCP,
-					Port:     int32(*ElPort),
-					TargetPort: intstr.IntOrString{
-						IntVal: int32(*ElPort),
-					},
-				},
+			Ports: []corev1.ServicePort{{
+				Name:     eventListenerServicePortName,
+				Protocol: corev1.ProtocolTCP,
+				Port:     int32(*ElPort),
+				TargetPort: intstr.IntOrString{
+					IntVal: int32(*ElPort),
+				}},
 			},
 		},
 	}
-	existingService, err := c.KubeClientSet.CoreV1().Services(el.Namespace).Get(el.Status.Configuration.GeneratedResourceName, metav1.GetOptions{})
+	existingService, err := r.KubeClientSet.CoreV1().Services(el.Namespace).Get(el.Status.Configuration.GeneratedResourceName, metav1.GetOptions{})
```

Comment:
this is a direct api call, you likely want to avoid doing this and use a lister and informer instead.

Reply:
fixed!